analysisdatalink/utils.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def position_column_to_matrix(col, voxel_size=np.array([4, 4, 40])):
```

Comment:
having `[4, 4, 40]` as default is counter intuitive given the name of this function. A user might _only_ expect the matrix output and not the scaling. It is a good feature to have in principle though.